docs/zh-CN/sql-reference/sql-functions/encrypt-digest-functions/sm4.md

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
You need to add these documents to the sidebar

Reply:
I have already add to side bar